.github/workflows/add-testlink-to-pr.yml

```diff
@@ -12,6 +12,6 @@ jobs:
           head-branch-regex: '.+'
           body-template: |
 
-            [Test link](https://sys-map.dev.bgdi.ch/%headbranch%/index.html)
+            [Test link](https://sys-map.dev.bgdi.ch/tl/%headbranch%/index.html)

```

Comment:
if https://github.com/geoadmin/infra-terraform-bgdi-builder/pull/69#discussion_r1025153610 goes into action, this will also need to change